src/test/java/de/tum/in/www1/artemis/service/SubmissionServiceTest.java

```diff
@@ -158,16 +158,16 @@ private void getTutorSpecificCallsTutor1(Exercise exercise) {
         database.changeUser("tutor1");
         unassessedSubmissionCorrectionRound0Tutor1 = submissionService.getRandomAssessableSubmission(exercise, true, 0);
         unassessedSubmissionCorrectionRound1Tutor1 = submissionService.getRandomAssessableSubmission(exercise, true, 1);
-        submissionListTutor1CorrectionRound0 = submissionService.getAllSubmissionsAssessedByTutorForCorrectionRoundAndExercise(exercise.getId(), tutor1, true, 0);
-        submissionListTutor1CorrectionRound1 = submissionService.getAllSubmissionsAssessedByTutorForCorrectionRoundAndExercise(exercise.getId(), tutor1, true, 1);
+        submissionListTutor1CorrectionRound0 = submissionService.getAllSubmissionsAssessedByTutorForCorrectionRoundAndExerciseIgnoreTestRuns(exercise.getId(), tutor1, true, 0);
```

Comment:
I would prefer a shorter name like 
`getAllSubmissionsAssessedByTutorIgnoreTestRuns` as exerciseId and correction round are communicated through the parameters of the function

Reply:
We currently use "ForCorrectionRound" in multiple methods, so I'd like to keep it consistent.
We might want to think about a refactoring PR that takes care of all these occurrences